calculation/sha_calc/sha_calc/directivity/bea20/benchmark_tests.py

```diff
@@ -0,0 +1,93 @@
+from pathlib import Path
```

Comment:
I might be missing something, but is this a script or a module? 
The `test_directivity` function doesn't appear to be used anywhere?